crates/relayer-cli/src/commands/clear.rs

```diff
@@ -75,6 +75,11 @@ impl Override<Config> for ClearPacketsCmd {
         })?;
 
         if let Some(ref key_name) = self.key_name {
+            // Q: should the key name be required across chain types, meaning that
+            // key management is common to all chain types, or should key management
+            // be the responsibility of the backend? If key management is common
+            // across backends, how should it be agnostic to the key type? Can it
+            // just be an opaque byte string handled by the backend?
```

Comment:
This ties in to https://github.com/informalsystems/hermes/issues/3641 

Perhaps the key manager should only support opaque byte strings, and let the rest be handled by the backend?